Bluepill-cli/Bluepill-cli/Bluepill/Bluepill.m

```diff
@@ -460,7 +460,9 @@ - (void)connectTestBundleAndTestDaemonWithContext:(BPExecutionContext *)context
     [bConnection connectWithTimeout:30];
```

Comment:
Also check return value here? Any other `connectWithTimeout` that need return checks?

Reply:
That's good idea before it cause problem. No other place has connectWithTimeout.